nss-day-cohort-30/bangazon-site-suave-salmon
# product images

Changes proposed in this request & reasons behind organizational or architectural decisions:
- product images can be uploaded on product create form
- images can be seen in product details and search results

Steps to test:
- git fetch --all
- git checkout kc-branch
- git pull origin kc-branch
- run program
- add new product, include image
- go to product details page for new item to see image displayed
- search products for your new product, image should be shown on results page

Commit message:
- product images in details and search

Link to feature ticket:
- [#17](https://github.com/nss-day-cohort-30/bangazon-site-suave-salmon/issues/17), [#19](https://github.com/nss-day-cohort-30/bangazon-site-suave-salmon/issues/19)

@suave-salmon
